msg/vehicle_status.msg

```diff
@@ -67,7 +74,7 @@ bool high_latency_data_link_active 		# all low latency datalinks to GCS lost
 uint8 data_link_lost_counter			# counts unique data link lost events
 bool engine_failure				# Set to true if an engine failure is detected
 bool mission_failure				# Set to true if mission could not continue/finish
-bool attitude_failure 				# Set to true if attitude failure has been detected
+uint8 failure_detector_status			# Bitmask containing FailureDetector status [0, 0, 0, 0, 0, FAILURE_ALT, FAILURE_PITCH, FAILURE_ROLL]
```

Comment:
This is fine, but you can feel free to use bools here if you want. The message is low rate so the space savings are negligible. For code though it requires more work to get or set a bit, and then for log analysis it ends up being a mess because the constants are lost. 

For example estimator_status bitfields (https://github.com/PX4/Firmware/blob/master/msg/estimator_status.msg#L25) ended up being a poor choice. These constants have been manually duplicated across both ecl and Firmware, the ecl analysis python scripts under Tools, in Flight review, etc. They're also fairly inaccessible with flightplot.

tl;dr - booleans + publish on change is your friend